Sources/SwiftPhoenixClient/Socket.swift

```diff
@@ -129,15 +110,15 @@ public class Socket: PhoenixTransportDelegate {
   public var enabledSSLCipherSuites: [SSLCipherSuite]?
   #endif
   
+  /// Collection on channels created for the Socket
+  public var channels: [Channel] = []
```

Comment:
Is `public func remove(_ channel: Channel)` not enough? Note it has a logic for disconnecting callbacks.

I'm worried exposing var for outside use will break internal mechanisms.

Reply:
Another option would be `public var hasOpenChannels: Bool { channels.isEmpty == false }`

Reply:
Totally fine with doing that so its clear that the channel array should't be modified